daemon/volumes.go

```diff
@@ -385,15 +385,14 @@ func copyExistingContents(source, destination string) error {
 // copyOwnership copies the permissions and uid:gid of the source file
 // into the destination file
 func copyOwnership(source, destination string) error {
-	var stat syscall.Stat_t
-
-	if err := syscall.Stat(source, &stat); err != nil {
+	stat, err := system.GetStat(source)
```

Comment:
@gewoonrik I confirm that in this case, stat and lstat are equivalent, I doublechecked the codepath. So let's just use system.Lstat here! Thanks @cpuguy83 for looking into this!